validator/rules_src/required_if.py

```diff
@@ -3,21 +3,19 @@
 
 class RequiredIf(Required):
     """
-    >>> TrueRequiredIf = RequiredIf('a')
-    >>> TrueRequiredIf('abc')
+    >>> RequiredIf('a').check('abc')
     True
 
-    >>> FalseRequiredIf = RequiredIf('z')
-    >>> FalseRequiredIf('abc')
+    >>> RequiredIf('z').check('abc')
     False
     """
 
     def __init__(self, requirement):
         Required.__init__(self)
         self.requirement = requirement
 
-    def __call__(self, arg):
-        if Required.__call__(self, arg) and self.requirement in arg:
+    def check(self, arg):
+        if super().check(arg) and self.requirement in arg:
```

Comment:
please change `super()` to `Required` (like in Between rule, we use names instead of calling super)